mini.go

```
@@ -116,10 +130,25 @@ func Poster(w http.ResponseWriter, r *http.Request) {
 		id, point.AdvertiserID, point.SiteID, ip, point.IosIfa)
 
 	if errs != nil {
-		fmt.Println(errs)
+		errString := fmt.Sprintf("bad insertion: %s", errs)
+		reponse(w, http.StatusBadRequest, errString)
```

This will always be a server-side error (if you validate your input properly), so it should be an `http.StatusInternalServerError`.
Also, you should hide the real error here just in case it contains some sensitive information. Just return a generic "There was an error." message.